client/tokenized-express-checkout/event-handlers.js

```diff
@@ -84,30 +110,39 @@ export const onConfirmHandler = async (
 
 	try {
 		// Kick off checkout processing step.
-		let orderResponse;
-		if ( ! order ) {
-			orderResponse = await api.expressCheckoutECECreateOrder(
-				normalizeOrderData( event, paymentMethod.id )
-			);
-		} else {
-			orderResponse = await api.expressCheckoutECEPayForOrder(
-				order,
-				normalizePayForOrderData( event, paymentMethod.id )
-			);
-		}
```

Comment:
No longer need to make a differentiation between pay-for-order and non-pay-for-order flow - the `cartApi` will take care of it.